docs/admin/troubleshooting.md

```diff
@@ -919,6 +919,46 @@ pool header correct
 0
 ```
 
+## Syslog
+
+[`RAS events`](https://docs.daos.io/v2.2/admin/administration/#ras-events) are printed to the Syslog
+by 'daos_server' processes via the Go standard library API.
+If no Syslog daemon is configured on the host, errors will be printed to the 'daos_server' log file:
+
+```
+ERROR: failed to create syslogger with priority %!s(syslog.Priority=24): Unix syslog delivery error
```

Comment:
Hm, the %!s stuff means there is no String() method on the type. IMO we should fix the formatting and update this message accordingly.

Reply:
will do, it will no longer be a doc change but I suppose that's okay

Reply:
done